pymedphys/tests/trf/test_decode.py

```diff
@@ -60,51 +44,56 @@ def compare_reference_to_converted(reference_dataframe, converted_dataframe):
         for column in reference_dataframe.columns:
             np.testing.assert_array_equal(
                 reference_dataframe[column].values,
-                reference_dataframe[column].values,
+                converted_dataframe[column].values,
```

Comment:
Here is quite an embarrassing find. So, not trusting my 2018 self (who was just learning how to write tests...) I decided to edit one of the columns within the baseline csv files to verify that the test would fail appropriately.

...It didn't fail, it still passed :(. Turns out, all this time, trf tests have just been comparing "reference" to "reference". Not "reference" to "converted". `*sigh*`

Reply:
Great find, this kind of thing happens to all of us from time to time :D

Reply:
> Great find, this kind of thing happens to all of us from time to time :D

Yeah, hopefully the good process we build up over time make this happen less and less though.